### DISCUSSION TITLE
Can't define modified/created with add-pnpfile

### DISCUSSION THREAD

**Discussion by glasserre-pdw** [pnp/powershell#694] [General]
Hello.
I tried to merge some SP libraries into a common library.
I have an issue to set modified/created date values with add-pnpfile

I tried : 

- like in the documentation .
Add-PnPFile -Path .\sample.doc -Folder "Shared Documents" -Values @{**Modified="1/1/2016"**}

- On the non .Net version this one was working
Add-PnPFile -Path .\sample.doc -Folder "Shared Documents" -Values @{**Modified="2016-01-01T10:30:00-00:00"**}

Do you have an idea of the correct format to used ?

Best regards.

**Comment by veronicageek** [comment]
@glasserre - Have a look at issue #627 which has been fixed
